content/post/mv-reflection-002.md

```diff
@@ -0,0 +1,44 @@
++++
+title = 'Co-ops for the rest of us'
+date = 2024-06-20T10:28:30-04:00
+authors = ["Max Veytsman"]
++++
+
+As part of my research, I started reading Jessica Livingston's [Founders at Work](http://www.foundersatwork.com/) to get a feel for the overculture.
+
+I chose the word overculture deliberately, but It's interesting to think that this book came out in 2008, two yeats before The Social Network. Back then, he startup world was not the "overculture" it was an underdog, and in fact the foreword (written by Paul Graham of course) is largely an argument for how the big businesses could stand to learn from the startupy way of doing things. How times have changed.
```

Comment:
Yeah I think about this a lot as someone with some instincts to "stick it to the man" but who has perhaps become The Man. There is definitely a pattern where someone who is historically the underdog comes into power but has a certain kind of chip-on-their-shoulder vibe that makes it hard for them to accept the different responsibility and social place they are now in. Certainly true of startup culture. Certainly true of "nerd culture". True of like most national founding myths. Most revolutions. Resistance to capital gains tax. Etc.

One practice I try to play with is to think of myself as a "wealthy philanthropist" now, rather than at some vague future magical wealth date. That is: wealth is extremely relative, and I see so much tendency for folks to only compare themselves "upwards" on the socio-economic ladder and always believe that they're struggling without tuning into the other perspective in which they're actually stupendously wealthy on any global perspective. At its weakest this just turns into a self-flagellatory "checking one's privilege" without any associated action, but at its best it reminds me that while maybe I can't name a hospital wing after myself perhaps I can give out a microgrant for a community project.